src/OrchardCore.Modules/Orchard.Autoroute/Drivers/AutoroutePartDisplay.cs

```diff
@@ -61,14 +65,16 @@ public override IDisplayResult Edit(AutoroutePart autoroutePart)
         public override async Task<IDisplayResult> UpdateAsync(AutoroutePart model, IUpdateModel updater)
         {
             await updater.TryUpdateModelAsync(model, Prefix, t => t.Path);
-            
+
             var httpContext = _httpContextAccessor.HttpContext;
-            
             if (httpContext != null && await _authorizationService.AuthorizeAsync(httpContext.User, Permissions.SetHomepage))
             {
                 await updater.TryUpdateModelAsync(model, Prefix, t => t.SetHomepage);
             }
 
+            var validationErrorKeyPrefix = string.IsNullOrEmpty(Prefix) ? string.Empty : Prefix + ".";
```

Comment:
Or because the updater is already "prefixed" (custom implementation) it could handle that by itself.